schoolyourself/schoolyourself_review.py

```diff
@@ -125,18 +125,18 @@ def handle_grade_json(self, data):
 
       # Verify the signature.
       if isinstance(self.shared_key, text_type):
-          self.shared_key = self.shared_key.encode('utf-8')
-      verifier = hmac.new(str(self.shared_key), user_id)
+        sk = self.shared_key.encode('utf-8')
+      verifier = hmac.new(sk, user_id.encode('utf-8'), digestmod='MD5')
```

Comment:
I think that `sk` will be undefined at this point if it does not enter in the previous if

Reply:
@morenol I fixed it assigning it before the if statement. Is it okay ?